### PR TITLE
ci: Run Wasm WinUI runtime tests

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Build or CI related changes

## What is the new behavior?

Run WinUI validations for WebAssembly.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]
Nice 👍

File: build/ci/.azure-devops-wasm-uitests.yml [hunk 1]
```diff
@@ -147,22 +154,43 @@ jobs:
...
+        SITE_SUFFIX: 'net7.0-UWP'
         UITEST_AUTOMATED_GROUP: 'RuntimeTests'
         UITEST_TEST_TIMEOUT: '1800000'
         UITEST_RUNTIME_TEST_GROUP: 1
         UITEST_RUNTIME_TEST_GROUP_COUNT: 3
 
-      net7_runtimetests_group_02:
-        SITE_SUFFIX: 'net7.0'
+      net7_uwp_runtimetests_group_02:
+        SITE_SUFFIX: 'net7.0-UWP'
```
Comment (MartinZikmund) [inline build/ci/.azure-devops-wasm-uitests.yml:172]: How does this determine which version of the source to run? I see `SITE_SUFFIX` used to differentiate between the artifacts, but not sure how the "uwp" group will pick UWP build and "winui" group will pick the converted one 🤔

Comment (MartinZikmund) [reply build/ci/.azure-devops-wasm-uitests.yml:172]: It really seems both UWP and WinUI Runtime tests are downloading the same build artifacts unless I am looking in the wrong place 😀

Comment (jeromelaban) [reply build/ci/.azure-devops-wasm-uitests.yml:172]: It could be :) I'll be fixing this once 4.6 is out :)

Comment (jeromelaban) [reply build/ci/.azure-devops-wasm-uitests.yml:172]: Thinking about it, there's no need for conversion here, as the tooling only uses already built samples.

**Pull request merged**